clients/helpers.go

```diff
@@ -5,20 +5,17 @@ import (
 	"io/ioutil"
 	"os"
 	"strings"
-
-	"github.com/urfave/cli/v2"
 )
 
 // ReadQuery reads a Flux query into memory from a --file argument, args, or stdin
-func ReadQuery(ctx *cli.Context) (string, error) {
-	nargs := ctx.NArg()
-	file := ctx.String("file")
+func ReadQuery(filepath string, args []string) (string, error) {
```

Comment:
The changes here are to allow kapacitor to upgrade `influx-cli` without needing to downgrade `urfave/cli` from v2 to v1. This was the only function that broke when I tried an upgrade locally.

Reply:
Hmm can you help me out a bit on what this means? Since this PR is downgrading from v2 to v1 I'm not sure I follow. Is it because kapacitor uses `urfave/cli` v2?

Reply:
Kapacitor imports `influx-cli` as a dependency so it can reuse the business logic under `clients/` (including this function). It also implements its own CLI front-end for the tasks, using `urfave/cli/v2`. Downgrading the module in `influx-cli` changed the type of the `cli.Context` here, breaking the calls in Kapacitor. This change is still breaking, but it's easier to work around than the alternatives of either downgrading urfave or somehow using both versions there.

I'd been trying to keep any references to `urfave/cli` out of the `clients/` package to give us more flexibility in situations like this. This one slipped through the cracks, so I'm glad to refactor it away.